etc/ncp-config.d/l10n/nc-backup-auto/de.json

```
@@ -1,12 +1,12 @@
 {
-    "translations":{
-    "ACTIVE":"Aktiviert",
-        "DESTDIR":"Ziel-Verzeichnis",
-        "INCLUDEDATA":"Daten mit sichern",
-        "COMPRESS":"Komprimieren",
-        "BACKUPDAYS":"Sicherungs-Periode (in Tage)",
-        "BACKUPLIMIT":"Sicherungs-Limit",
-        "nc-backup-auto":"Automatische Sicherung"
-        
+    "translations": {
+        "nc-backup-auto": "Automatische Sicherung",
+        "ACTIVE": "Aktiv",
+        "DESTDIR": "Zielverzeichnis",
+        "INCLUDEDATA": "Daten inkludieren",
```

[Inkludieren](https://www.duden.de/synonyme/inkludieren) is not commonly used in German language.
```suggestion
        "INCLUDEDATA": "Daten ebenfalls sichern",
```